src/org/opendatakit/briefcase/util/TransferFromODK.java

```diff
@@ -241,13 +229,7 @@ public boolean accept(File dir, String name) {
                 int i = 2;
                 boolean same = false;
                 while (scratchInstance.exists()) {
-                  File[] contents = scratchInstance.listFiles(new FilenameFilter() {
-
-                    @Override
-                    public boolean accept(File dir, String name) {
-                      return name.endsWith(".xml");
-                    }
-                  });
+                  File[] contents = scratchInstance.listFiles((dir1, name) -> name.endsWith(".xml"));
```

Comment:
Change the lambda arg's name from `dir1` to `__`

Reply:
Also, the `(__, name) -> name.endsWith(".xml")` lambda is duplicated, extract to a constant member and give a proper name.

Reply:
I added this line

```java
private FilenameFilter xmlFilenameFilter = (__, name) -> name.endsWith(".xml");
```
And the checkstyle plugin gave this error 


[ant:checkstyle] [ERROR] /home/emajj/workspace/briefcase/src/org/opendatakit/briefcase/util/TransferFromODK.java:43:47: Parameter name '__' must match pattern '^[a-z]([a-z0-9][a-zA-Z0-9]*)?$'. [ParameterName]

Reply:
OK, let's add an exception to the rule here. Just annotate that line with this: 
```java
@SuppressWarnings("checkstyle:ParameterName")
```